Marlin/src/lcd/language/language_de.h

```diff
@@ -151,65 +154,66 @@
 #define MSG_MOTION                          _UxGT("Bewegung")
 #define MSG_FILAMENT                        _UxGT("Filament")
 #define MSG_VOLUMETRIC_ENABLED              _UxGT("E in mm³")
-#define MSG_FILAMENT_DIAM                   _UxGT("D Fil.")
+#define MSG_FILAMENT_DIAM                   _UxGT("Durchm. Filament")
 #define MSG_FILAMENT_UNLOAD                 _UxGT("Entladen mm")
 #define MSG_FILAMENT_LOAD                   _UxGT("Laden mm")
-#define MSG_ADVANCE_K                       _UxGT("Advance Faktor")
+#define MSG_ADVANCE_K                       _UxGT("Vorschuss Faktor")
 #define MSG_CONTRAST                        _UxGT("LCD Kontrast")
 #define MSG_STORE_EEPROM                    _UxGT("Konfig. speichern")
 #define MSG_LOAD_EEPROM                     _UxGT("Konfig. laden")
 #define MSG_RESTORE_FAILSAFE                _UxGT("Standardwerte laden")
 #define MSG_INIT_EEPROM                     _UxGT("Werkseinstellungen")
-#define MSG_SD_UPDATE                       _UxGT("SD Update")
-#define MSG_RESET_PRINTER                   _UxGT("Drucker neustarten")
 #define MSG_REFRESH                         _UxGT("Aktualisieren")
-#define MSG_WATCH                           _UxGT("Info")
+#define MSG_WATCH                           _UxGT("Infos")
 #define MSG_PREPARE                         _UxGT("Vorbereitung")
 #define MSG_TUNE                            _UxGT("Justierung")
-#define MSG_PAUSE_PRINT                     _UxGT("SD-Druck Pause")
-#define MSG_RESUME_PRINT                    _UxGT("SD-Druck Fortsetzung")
-#define MSG_STOP_PRINT                      _UxGT("SD-Druck Abbruch")
-#define MSG_CARD_MENU                       _UxGT("SD-Karte")
+#define MSG_PAUSE_PRINT                     _UxGT("SD-Druck Pausieren")
+#define MSG_RESUME_PRINT                    _UxGT("SD-Druck Fortsetzen")
+#define MSG_STOP_PRINT                      _UxGT("SD-Druck Abbrechen")
+#define MSG_POWER_LOSS_RECOVERY             _UxGT("Wiederh. n. Stroma.")
+#define MSG_CARD_MENU                       _UxGT("Druck v. SD-Karte")
 #define MSG_NO_CARD                         _UxGT("Keine SD-Karte")
 #define MSG_DWELL                           _UxGT("Warten...")
-#define MSG_USERWAIT                        _UxGT("Warte auf Nutzer")
+#define MSG_USERWAIT                        _UxGT("Klick zum fortsetzen")
 #define MSG_PRINT_PAUSED                    _UxGT("Druck pausiert")
+#define MSG_PRINTING                        _UxGT("Druckt...")
 #define MSG_PRINT_ABORTED                   _UxGT("Druck abgebrochen")
-#define MSG_NO_MOVE                         _UxGT("Motoren eingeschaltet")
+#define MSG_NO_MOVE                         _UxGT("Motoren einschalten")
```

Comment:
Depends on what is the context. If its a status then "eingeschaltet" if its an action then "einschalten"

Reply:
It being used as kind of feedback after "M17" (Enable Steppers).

/**
 * M17: Enable power on all stepper motors
 */
void GcodeSuite::M17() {
  LCD_MESSAGEPGM(MSG_NO_MOVE);
  enable_all_steppers();
}

so "eingeschaltet" is more like a feedback (e.g. "steppers have been enabled")

Reply:
oh sorry misunderstanding from me 😥